src/styles/styles.js

```diff
@@ -601,6 +601,11 @@ const styles = {
         color: themeColors.heading,
     },
 
+    createMenuDescription: {
+        fontSize: variables.fontSizeLabel,
+        color: colors.gray4,
```

Comment:
Could you use something more descriptive like `color: themeColors.textSupporting,`?

Reply:
Done!